lua/org-bullets.lua

```diff
@@ -6,9 +6,9 @@ local NAMESPACE = api.nvim_create_namespace("org-bullets")
 local org_headline_hl = "OrgHeadlineLevel"
 
 local list_groups = {
-  ["-"] = "OrgHeadlineLevel1",
-  ["+"] = "OrgHeadlineLevel2",
-  ["*"] = "OrgHeadlineLevel3",
+  ["-"] = "OrgBulletsDash",
```

Comment:
@hisbaan where are these new groups being aliased to the original ones? I'd expect see something like
```vim
highlight link OrgBulletsDash OrgHeadlineLevel1
```
NOTE: this is vimscript so would actually expect the lua equivalent

Reply:
Ah, good point! I've added the appropriate links with the lua api